reflex/utils/serializers.py

```diff
@@ -319,7 +319,7 @@ def serialize_image(image: Img) -> str:
         image.save(buff, format=getattr(image, "format", None) or "PNG")
         image_bytes = buff.getvalue()
         base64_image = base64.b64encode(image_bytes).decode("utf-8")
-        mime_type = getattr(image, "get_format_mimetype", lambda: "image/png")()
+        mime_type = getattr(image, "get_format", lambda: "image/png")()
```

Comment:
i don't think `get_format` is actually valid here... when i examine the resulting mime_type, it comes out as `image/png`, even thought the image is actually a jpeg